docs/vaahcms-2/backend/roles.md

```diff
@@ -0,0 +1,238 @@
+# Roles
+
+[[toc]]
+
+## Purpose
+A role is a group of permissions that allow a user who has been given the role to carry out specific tasks. For instance, the super administrator, who can be regarded as the application's owner, is permitted to carry out all tasks in the application. He or she can add users, remove or edit, and do other things.
```

Comment:
@wri-tanmoy-d001 @we-vikram-wri231 

- [ ] `specific tasks` -> `specific actions`
- [ ] `He or she` -> `They`
- [ ] Give example of one more `role` that can do less, write what they can't do

Reply:
@themodernpk Ok Sir,

we will update all these changes